graphsdk/src/main/java/com/microsoft/graph/generated/BaseDirectoryObjectReferenceRequest.java

```diff
@@ -38,7 +38,7 @@ public void delete(final ICallback<DirectoryObject> callback) {
     }
 
     public DirectoryObject delete() throws ClientException {
-       return send(HttpMethod.DELETE, null);
+        return send(HttpMethod.DELETE, null);
```

Comment:
Appears to be an artifact of an existing formatting error in the template. I don't stress this stuff, but it's there.